lib/bone.js

```diff
@@ -83,6 +83,14 @@ class Bone {
       rawInitial: {},
       rawUnset: new Set(),
     }));
+
+    Object.defineProperty(this, 'isNewRecord', {
+      value: true,
+      enumerable: false,
+      writable: true,
+      configurable: true,
+    });
```

Comment:
A direct assignment, such as `this.isNewRecord = true`, shall suffice.

Reply:
btw, I did take a look in Active Record of Ruby on Rails. The equivalent property is called `new_record?` (notice the `?` at the end, which indicates it's a boolean). Hence I think `isNewRecord` is perfect for us.

Reply:
`isNewRecord` should not be  enumerable, a direct assignment will make developer confuse

Reply:
good point.